devito/passes/iet/linearization.py

```diff
@@ -20,138 +22,148 @@
 def linearize(graph, **kwargs):
     """
     Turn n-dimensional Indexeds into 1-dimensional Indexed with suitable index
-    access function, such as `a[i, j]` -> `a[i*n + j]`. The row-major format
+    access function, such as `a[i, j] -> a[i*n + j]`. The row-major format
```

Comment:
Stupid question, Can this be bypassed into column major? Could be helpful for like vx since julia arrays are column majors then the "ExternalAllocator" could be really powerful.

But side thought can ignore

Reply:
not sure I get what u mean by "bypassed"?

I think we can tweak linearization to switch to column-major indexing (pending user-provided options)

FIndexed is agnostic w.r.t. the actual indexing strategy. It just uses whatever the user has passed to instantiate it